proposals/000-comm-grants.md

```diff
@@ -0,0 +1,223 @@
+the good of the entire Haskell community.  This indefinite ongoing program would
+accept submissions from the community on a rolling basis, and award up to $2000
+grants to individuals as a stipend to support their work on the proposed
+project.
+
+The amount of money involved is small, so the primary risk of the proposal is
+that funding decisions would generate discouragement or ill will from rejected
+submissions.  The risk can be partially mitigated by the use of a random lottery
+system to select winners after an initial selection of submissions that the
+Foundation believes are worth accepting.
+
+## Background
+
+Numerous mechanisms currently exist for individuals to financially support parts
+of the Haskell community, including projects that have individually set up
+fundraising, individuals who have requested donations via Patreon, GitHub
+sponsorships, etc., and purchase of services and subscriptions to content.  Each
+of these choices supports a specific part of the Haskell community, and none of
+them offer a default way to contribute to Haskell as a whole.
+
+The Haskell Foundation now also accepts individual donations, and its
+organizational mission is consistent with being precisely such an appealing
+target for general-purpose donations to Haskell.  However, there is currently
+not a clear picture for how funds make it from donations to the Haskell
+Foundation to directly encouraging contributions to the Haskell community.  In
+surveys of the community, individuals who have decided not to contribute to the
+Haskell Foundation have given the reason that they don't see that their
+contributions would support the Haskell open source community.
+
+The HFTP process superficially appears to offer such a path.  However, there are
+several barriers to its being used directly by individuals or small projects for
+which funding could make a difference.  Note that "individuals and small
+projects" describes a vast portion of crucial Haskell libraries and tools.
+
+1. The HFTP process is not encouraging to individuals looking for small amounts
+   of funding, and says things like "Typically, this includes many rounds of
+   discussion with HF leaders, Haskell library maintainers, and the broader
+   community, several iterations on the design of the proposal, and some effort
+   at prototyping the proposed change."
+2. The benefit of small grants can depend on life circumstances, such as one's
+   need for child care, willingness to leave an employer, etc.  Individuals may
+   prefer not to include this information in a document open for public comment.
+
+Thus, the HFTP process isn't a great fit for direct requests from individuals or
+small projects for financial support.  Furthermore, the HFTP process gives an
+explicit example of a "Summer of Code" style program as within scope for an HFTP
+proposal.  This is such a proposal.
+
+Several other communities have similar grant programs.  Some examples include:
```

Comment:
I would love to know more about how these programs work in other communities -- especially the open source ones. (While I do know the internals about the Tweag Fellowship program, I think that program is sufficiently different from what's being proposed here so as to make our data irrelevant.) What are those programs' goals? Have the programs achieved the goals? What's the role of these programs within their communities?